content/en/docs/papers/fsep/_index.md

```diff
@@ -0,0 +1,218 @@
+
+The decentralized space has wiped this slate clean. And while the openness of the fediverse is a massive step forward in independent networks becoming an everyday experience on the web, the downside is that many of these spaces have become unregulated with little to no moderation at all. 
+
+These unregulated [spaces](https://theconversation.com/citizens-social-media-like-mastodon-can-provide-an-antidote-to-propaganda-and-disinformation-192491) attract hateful and aggressive people looking to shake off the restrictions by centralized platforms to let their bigotry fly without fear of consequences as the larger fediverse looks to find its feet. And unfortunately, this lack of attention can result in [monocultures](https://techpolicy.press/the-whiteness-of-mastodon/) that can have (content warning -  violence) [devastating](https://www.wired.com/story/pittsburgh-synagogue-shooting-gab-tree-of-life/) consequences. 
+
+One convention that federated platforms have adopted to combat harassment and abuse has been blocklists, which enable sites to refuse to federate with any  URL deemed a threat to said site. The now-defunct [Block Together](https://blocktogether.org/) used a similar methodology to enhance safety and abuse mitigation on Twitter.  
+
+The obvious issue is that every site can have its blocklists, which is excellent for individual setups. However, this makes collaboration between these lists challenging, which can hamper coordination around dealing with abusive sites that perpetually engage in harassment and bad faith engagement. 
+
+To prioritize this [collaboration](https://codeberg.org/oliphant/blocklists) and make it easier to defederate with sites with a  history of poor behavior, The Bad Space was created, a searchable database populated with sites that multiple instances have blocked. While still in alpha, this site provides new opportunities to enhance safety and abuse mitigation and improve onboarding and blocklist maintenance by providing an easy way to keep blocklists up to date from a central location through dynamically created export lists and public API that can be used to search the database programmatically.
+
+These features can be integrated into existing platforms to enhance the overall experience on Activity Pub-enabled applications by automatically limiting the opportunity for bad-faith actors to interact.  
+
+
+## Functional Requirements
+
+### Service Feature Integration 
+
+The first step in this process is to make defederated systems aware of the features provided by services such as The Bad Space, specifically its public search API and dynamically created and updated blocklists. 
```

Comment:
Did you mean "defederate" or perhaps "decentralised" or "federated" here?

Reply:
Oops, yes. Either works.